src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Queries/Types/TypedContentTypeBuilder.cs

```diff
@@ -59,12 +62,12 @@ public void Build(FieldType contentQuery, ContentTypeDefinition contentTypeDefin
                         return;
                     }
 
-                    var whereInput = (ContentItemWhereInput) whereArgument.ResolvedType;
+                    var whereInput = (ContentItemWhereInput)whereArgument.ResolvedType;
 
                     whereInput.AddField(new FieldType
                     {
                         Type = inputGraphTypeResolved.GetType(),
-                        Name = partName.ToCamelCase(),
+                        Name = inputGraphTypeResolved.HasMetadata("DisplayName") ? inputGraphTypeResolved.GetMetadata<string>("DisplayName").ToCamelCase() : partName.ToCamelCase(),
```

Comment:
DisplayName and partName have the same value for the input parts. What does this extra code achieve currently?

Reply:
Only in Orchard's Core... this might change in other systems, so this give us the ability to override